lib/ethpy/ethpy/test_fixtures/deploy_hyperdrive.py

```diff
@@ -217,17 +217,20 @@ def deploy_and_initialize_hyperdrive(
         oracle_size,  # oracleSize
         update_gap,
     )
-    tx_receipt = smart_contract_transact(
-        web3,
-        factory_contract,
-        deploy_account,
-        "deployAndInitialize",
-        # Function arguments
+    # Function arguments
+    fn_args = (
```

Comment:
readability change, this allows me to verify the function because I can see which parameters are for the python function and which are passed to the solidity function (fn_args)